src/routes/components/Modal.tsx

```diff
@@ -223,17 +261,23 @@ export default function TEMPLATE_Page() {
                 code={`
                   const [open, setOpen] = useState(false);
                    function onClick() {
-                    setOpen(true);
+                    setOpen(prevOpen => !prevOpen);
```

Comment:
This line is fine as is, since the button only ever opens the modal

Reply:
Done